apps/ledger-live-mobile/src/screens/Onboarding/shared/infoPagesData.tsx

```diff
@@ -470,7 +470,10 @@ export const infoModalScenes = {
   pairNewErrorInfoModalProps,
 };
 
-const pinCodeScenes = (deviceModelId, theme: "dark" | "light") => [
+const pinCodeScenes = (
+  deviceModelId: "nanoS" | "nanoSP" | "nanoX" | "blue" | "nanoFTS",
```

Comment:
Safer to use the enum `DeviceModelId` from:
```TS
import { DeviceModelId } from "@ledgerhq/types-devices";
```

Reply:
nevermind it will be a work for the device team i guess

Reply:
yep 😄